frontend/packages/console-shared/src/hooks/useUserSettingsCompatibility.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { useUserSettings } from './useUserSettings';
+
+export const useUserSettingsCompatibility = <T>(
+  key: string,
+  storageKey: string,
+  defaultValue?: T,
+) => {
+  const [settings, setSettings, loaded] = useUserSettings(
+    key,
+    localStorage.getItem(storageKey) || defaultValue,
+  );
+  const updaterFunction = React.useCallback(
+    (data) => {
+      setSettings(data);
+    },
+    [setSettings],
+  );
```

Comment:
no need for this, we can directly pass the setSettings

Reply:
yeah passed `setSettings`